packages/editor/templates/Coin/app/app.js

```diff
@@ -57,34 +59,44 @@ Coin.prototype.showAddressBalance = function(hash, cb) {
     })
 }
 
-// Send Tokens to another address
+// Get balance of Tokens found by address from contract
+Coin.prototype.getBalance = function(address, cb) {
+    this.instance.balances(address, function(error, result) {
+        cb(error, result);
+    })
+}
+
+// Send Tokens to another address when the "send" button is clicked
 Coin.prototype.createTokens = function() {
     var that = this;
 
-    // Get input values
+    // Get input values for address and amount
     var address = $("#create-address").val();
     var amount = $("#create-amount").val();
     console.log(amount);
 
-
-    // Validate address
+    // Validate address using utility function
     if(!isValidAddress(address)) {
         console.log("Invalid address");
         return;
     }
 
-    // Validate amount
+    // Validate amount using utility function
     if(!isValidAmount(amount)) {
         console.log("Invalid amount");
         return;
     }
 
     // Transfer amount to other address
+    // Use the public mint function from the smart contract
     this.instance.mint(address, amount, { from: window.web3.eth.accounts[0], gas: 100000, gasPrice: 100000, gasLimit: 100000 }, 
+        // If there's an erro, log it
```

Comment:
`error`

Reply:
```suggestion
        // If there's an error, log it
```

The keyboard strikes again!